index.html

```diff
@@ -171,11 +137,11 @@ <h3>Looks like your system can run Freckle, if you have trouble please send us a
       <footer class="row">
         <div class="col-xs-12 text-center uv-padding">
           <p>
-            <a target="_blank" href="https://frontrowed.zendesk.com">More information about Freckle</a>
+            <a target="_blank" href="https://freckle.zendesk.com">More information about Freckle</a>
```

Comment:
Can I recommend using the official CNAME for it, so that schools don't have to whitelist zendesk? https://support.freckle.com

Reply:
Can do